bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/thing/channel-group-types.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" ?>
+<thing:thing-descriptions bindingId="wundergroundupdatereceiver"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="metadata-group">
+		<label>Metadata</label>
+		<description>Metadata regarding the last submission</description>
+		<category>Text</category>
+		<channels>
+			<channel id="last-received" typeId="last-received-datetime"/>
+			<channel id="dateutc" typeId="datetime-utc"/>
+			<channel id="softwaretype" typeId="softwaretype"/>
+			<channel id="last-query-state" typeId="last-query-state"/>
+			<channel id="last-query-trigger" typeId="last-query-trigger"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="wind-group">
+		<label>Wind</label>
+		<description>Wind measurements</description>
+		<category>Wind</category>
+		<channels>
+			<channel id="winddir" typeId="system.wind-direction"/>
+			<channel id="windspeedmph" typeId="wind-speed"/>
+			<channel id="windgustmph" typeId="wind-gust-speed"/>
+			<channel id="windgustdir" typeId="wind-gust-direction"/>
+			<channel id="windspdmph_avg2m" typeId="wind-speed-avg-2min"/>
+			<channel id="winddir_avg2m" typeId="wind-direction-avg-2min"/>
+			<channel id="windgustmph_10m" typeId="wind-gust-speed-10min"/>
+			<channel id="windgustdir_10m" typeId="wind-gust-direction-10min"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="temperature-group">
+		<label>Temperature</label>
+		<description>Temperature measurements</description>
+		<category>Temperature</category>
+		<channels>
+			<channel id="tempf" typeId="system.outdoor-temperature"/>
```

Comment:
is it possible to calculate heatindex as well?
[heatindex_equation.shtml](https://www.wpc.ncep.noaa.gov/html/heatindex_equation.shtml)

Reply:
See below